src/components/App.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  CssBaseline,
+  createStyles
+} from '@material-ui/core';
+import { FolderList } from './FolderList';
+import { useFolderList } from '../hooks/useFolderList';
+
```

Comment:
sort `import` lines

Reply:
I usually prefer to sort my imports myself. I tend to put react first, then any other * imports, then any desctuctured node_module imports and finally my own modules import. But if you want to sort them alphabetically, we could give it a try.

Reply:
That is also my preference. What I've learned though, is that it is my opinion and it is not _easy_ to enforce the rule in a decisive manner for all members of a team.

The only _logical_ and _unopiniated_ strategy that I have experience and that didn't brought a long conversation about the order of imported elements is the alphabetical order.

It is an order, when you search for an import: your brain knows quickly where it should be, there is only one order possible and it is really easy to _fix_ in any text editor or IDE that is worth mentioning.

I will not block this PR for such a small thing. We'll sort next time we touch the file.